packages/cli/src/extensions/aws-extension.ts

```diff
@@ -159,11 +209,46 @@ module.exports = (toolbox: GluegunToolbox) => {
   const regionChain = (...options: RegionOptions[]): Observable<string> =>
     createChain(options, region)
 
+  const tableName = (options?: TableNameOptions): Observable<string> => {
+    options = deepmerge(defaultTableNameOptions, options || {})
```

Comment:
Der nullish coalescing operator ist der, den du hier eigentlich verwenden willst
```suggestion
    options = deepmerge(defaultTableNameOptions, options ?? {})
```